libraries/ms-helper/rules.mk

```diff
@@ -7,3 +7,7 @@
 
 # Specify the libraries you want to include
 $(T)_DEPS := ms-common
+
+ifeq (x86,$(PLATFORM))
+$(T)_EXCLUDE_TESTS := thermistor
```

Comment:
You shouldn't need to exclude the test. You can mock out `adc_read_converted` to test different scenarios.

Reply:
Is there an example of this in the code base that I could use as a reference?

Reply:
Talk to @ericslchiang or look at `test_mock`. There are a bunch of examples in our tests.